packages/vaadin-date-picker/package.json

```diff
@@ -32,7 +32,6 @@
     "@vaadin/component-base": "^22.0.0-alpha6",
     "@vaadin/field-base": "^22.0.0-alpha6",
     "@vaadin/input-container": "^22.0.0-alpha6",
-    "@vaadin/text-field": "^22.0.0-alpha6",
```

Comment:
Seems that `@vaadin/vaadin-text-field` dependency could also be dropped (can be done in a separate PR as it doesn't directly relate to this particular change).

Reply:
Done.